tests/end2end/helpers/_datasets.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
I would rather name the file by `_datasets_generator.py`

Reply:
Thank you I will do that.

Reply:
... sorry I forgot why I used that name in the first place. It is a generator for now, but it can be the module where we can add all dataset download operations.